engine/runner/src/main/java/org/enso/runner/Main.java

```diff
@@ -933,7 +925,18 @@ private void runRepl(
                 .disableLinting(true)
                 .enableStaticAnalysis(enableStaticAnalysis)
                 .build());
-    var mainModule = context.evalModule(dummySourceToTriggerRepl, replModuleName);
+
+    var sb = new StringBuilder();
+    sb.append("import Standard.Base.Runtime.Debug\n");
+
+    for (var libName : context.getTopScope().getLibraries()) {
+      sb.append("from ").append(libName).append(" import all\n");
+    }
```

Comment:
won't we risk having clashing symbols in the libraries?
We probably shouldn't have but I don't think we have any test that ensures this.

Alternatively, we could do `import X.Y` - then the user would need to use FQN to access the imported symbols. What do you think?

Reply:
We don't have any conflicts right now, but you are the user...

Reply:
I think `import all` is more user-friendly, so long as we test somewhere that we don't introduce conflicts.

Reply:
Test to verify script compiles without conflicts provided in 78db664

Reply:
Thank you